strapi/strapi
# Feat(releases): Bulk Release

### What does it do?

- This adds the ability to add several entries to a release from list view.
- Bulk Actions modal is updated to accept directly content which will contain modal body and modal footer. Earlier having footer and content(body) as separate properties makes it difficult to have connection between body and footer.

### How to test it?

On list view, whenever user selects multiple entries, if D&P is enabled and user has `create release` permissions, user should be able to add bulk entries to a single release. [Designs](https://www.figma.com/file/7YA3cGhgS11lWffL44hsLJ/Releases?type=design&node-id=127-12047&mode=design&t=YJ8GeR6MWR2mN2yK-0)
